glide.yaml

```diff
@@ -45,7 +45,8 @@ import:
   version: v2.0.0
 - package: github.com/JiangtianLi/gettext
   version: a8983c062be4b565d723c478922d7736e04fdba4
-testImport:
+- package: github.com/kelseyhightower/envconfig
+  version: ~1.3.0
```

Comment:
FYI all new OSS components and version bumps of existing components need to be registered with Microsoft OSS. I registered this one: https://ossmsft.visualstudio.com/_oss?requestId=44754&_a=progress